prometheus_metrics_latest.json

```diff
@@ -0,0 +1,306 @@
+{
+    "generatedWith": "parse_prometheus_metrics.py",
+    "info": "Recreate by extracting output in kuttl test 'kubectl test --test metrics-no-auth'",
+    "metrics": [
+        {
+            "description": "Number of currently used workers per controller",
```

Comment:
Add a period to the end of each description.

Reply:
Some of these we don't have control over the text. But I was able to add a period at the end of each description in the script that pulls the metrics.